### ISSUE TITLE
Ubuntu 14:04 Compile Error

### ISSUE THREAD

**Issue by sunatthegilddotcom** [google/guetzli#90]
501  sudo apt-get install libpng-dev libgflags-dev
  502  git clone https://github.com/google/guetzli.git
  503  cd guetzli/
 make
 make
==== Building guetzli (release) ====
Linking guetzli
/bin/sh: 1: pkg-config: not found
/bin/sh: 1: pkg-config: not found
obj/Release/guetzli.o: In function `ReadPNG':
/home/wsun/guetzli/guetzli/guetzli.cc:52: undefined reference to `png_create_read_struct'
/home/wsun/guetzli/guetzli/guetzli.cc:57: undefined reference to `png_create_info_struct'
/home/wsun/guetzli/guetzli/guetzli.cc:70: undefined reference to `png_init_io'
/home/wsun/guetzli/guetzli/guetzli.cc:80: undefined reference to `png_read_png'
/home/wsun/guetzli/guetzli/guetzli.cc:82: undefined reference to `png_get_rows'
/home/wsun/guetzli/guetzli/guetzli.cc:84: undefined reference to `png_get_image_width'
/home/wsun/guetzli/guetzli/guetzli.cc:85: undefined reference to `png_get_image_height'
/home/wsun/guetzli/guetzli/guetzli.cc:88: undefined reference to `png_get_channels'
/home/wsun/guetzli/guetzli/guetzli.cc:145: undefined reference to `png_destroy_read_struct'
/home/wsun/guetzli/guetzli/guetzli.cc:142: undefined reference to `png_destroy_read_struct'
/home/wsun/guetzli/guetzli/guetzli.cc:59: undefined reference to `png_destroy_read_struct'
obj/Release/guetzli.o: In function `main':
/home/wsun/guetzli/guetzli/guetzli.cc:195: undefined reference to `google::SetUsageMessage(std::string const&)'
/home/wsun/guetzli/guetzli/guetzli.cc:196: undefined reference to `google::ParseCommandLineFlags(int*, char***, bool)'
/home/wsun/guetzli/guetzli/guetzli.cc:199: undefined reference to `google::ShowUsageWithFlags(char const*)'
obj/Release/guetzli.o: In function `__static_initialization_and_destruction_0':
/home/wsun/guetzli/guetzli/guetzli.cc:38: undefined reference to `google::FlagRegisterer::FlagRegisterer(char const*, char const*, char const*, char const*, void*, void*)'
/home/wsun/guetzli/guetzli/guetzli.cc:40: undefined reference to `google::FlagRegisterer::FlagRegisterer(char const*, char const*, char const*, char const*, void*, void*)'
collect2: error: ld returned 1 exit status
make[1]: *** [bin/Release/guetzli] Error 1
make: *** [guetzli] Error 2

**Comment by david-poirier-csn** [comment]
Same issue on Ubuntu 16.04 and Centos 7, although no pkg-config errors.

**Comment by robryk** [comment]
@david-poirier-csn:
> Same issue on Ubuntu 16.04 and Centos 7, although no pkg-config errors.

This sounds then like a different issue. If this still persists after #97, please open a new issue and paste a complete log of the failing build.

**Comment by robryk** [comment]
@sunatthegilddotcom 

First, please try the current version (after #97 was merged). If that doesn't work, please try running `pkg-config`. If that command is not found, please see if installing as follows helps:

```
sudo apt-get install pkg-config
```

**Comment by david-poirier-csn** [comment]
Current version works for me now, on Ubuntu 16.04 and Centos 7.

**Comment by sunatthegilddotcom** [comment]
@david-poirier-csn your solution is working. Thank you very much!  @robryk, I followed your direction as well after David. Thank you

**Comment by JulianSSS** [comment]
Thanks, @robryk 
this solution works for me too!
